packages/filler/src/services/ContractInteractionService.ts

```diff
@@ -582,10 +589,15 @@ export class ContractInteractionService {
 	 */
 	async convertGasToFeeToken(gasEstimate: bigint, chain: string, targetDecimals: number): Promise<bigint> {
 		const client = this.clientManager.getPublicClient(chain)
+		const useEtherscan = chain.includes("137") || chain.includes("56") || chain.includes("1")
```

Comment:
Let's use an actual equaity check here instead of includes, for example if the chain is uniswap with chain id 130 or arbitrum with chain id 42161, the includes chain.includes("1") will succeed even though that's not what we want

Reply:
Ohh damn